### ISSUE TITLE
IEDriverServer Crashing With Selenium 3.13 on Windows 7

### ISSUE THREAD

**Issue by TARIFABBAS** [SeleniumHQ/selenium#6192]
## Meta -
OS:  Windows 7

Selenium Version:  Selenium 3.13

Browser:  Internet Explorer

Browser Version:  11.0.9600.1908015 (64 Bit) and 10.0.9200.17609 (64 Bit)

Driver : IEDriverServer 3.12; IEDriverServer 3.13

## Expected Behavior - 
Performing some steps on [this ](https://glip.com/hp_b5)url

## Actual Behavior -
While performing click IEDriver is getting crash.
Soetimes it is giving Element not clickable exception.

## Steps to reproduce -
Steps ::
1> open [This](https://glip.com/hp_b5)
2> Click on "Product"
3> Click on "Facebook link Icon" situated on the Bottom right corner on the page.
4> Exception occurs while clicking on the obove link and IEDriver crashing.

Please find below code to reproduce the problem::
```
DesiredCapabilities capabilities = DesiredCapabilities.internetExplorer();
		capabilities.setCapability("native events", true);
		// capabilities.setCapability("requireWindowFocus", true);
		capabilities.setCapability("ignoreProtectedModeSettings", true);
		capabilities.setCapability("unexpectedAlertBehaviour", "ignore");
		capabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
		capabilities.setCapability("handlesAlerts", false);

		WebDriver driver = new InternetExplorerDriver(capabilities);

		driver.get("https://glip.com/hp_b5");
		System.out.println("Title is as :: " + driver.getTitle());

		// click on product
		driver.findElements(By.xpath("/html/body/div[2]/header/div/nav/ul/li[1]/a")).get(0).click();
		if (args.length > 1)
			Thread.sleep(5000);

		// click on facebook 
               // Crashing At This Time
		// driver.findElements(By.xpath("/html/body/div[2]/footer/div/div[2]/ul/li[1]")).get(0).click();
		driver.findElement(By.xpath("/html/body/div[2]/footer/div/div[2]/ul/li[1]")).click();

		Thread.sleep(5000);

		// click on pricing
		List<WebElement> ele = driver.findElements(By.xpath("/html/body/div[2]/header/div/nav/ul/li[2]/a"));
		System.out.println(ele.size());
		ele.get(0).click();

		// JavascriptExecutor jse = (JavascriptExecutor) driver;
		// jse.executeScript("arguments[0].click();", ele.get(0));

		Thread.sleep(15000);
		driver.close();
		driver.quit();
```

**It is working fine with both driver on windows 10**

**Comment by jimevans** [comment]
@TARIFABBAS Can you please retry this repro with the `IEDriverServer.exe` version 3.13.0.3 [checked into the source tree](https://github.com/SeleniumHQ/selenium/blob/master/cpp/prebuilt/Win32/Release/IEDriverServer.exe)?

**Comment by jimevans** [comment]
@TARIFABBAS Contrary to the initial issue report, the stack trace you just posted seems to indicate you’re using 3.4 of Selenium with 3.13.0.3 of the IE driver. You’ll need to update the Selenium Java library for a valid test.

**Comment by TARIFABBAS** [comment]
Hi @jimevans Same is happening with latest Selenium
Please find the logs below:
`Started InternetExplorerDriver server (32-bit)
3.13.0.3
Listening on port 6489
Only local connections are allowed
Jul 25, 2018 2:15:06 PM org.openqa.selenium.remote.ProtocolHandshake createSessi
on
INFO: Detected dialect: W3C
Title is as :: Team Collaboration Software | RingCentral Glip
Exception in thread "main" org.openqa.selenium.WebDriverException: java.net.Sock
etException: Connection reset
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:19.8
91Z'
System info: host: 'CTND1147', ip: '172.17.20.28', os.name: 'Windows 7', os.arch
: 'amd64', os.version: '6.1', java.version: '1.8.0_131'
Driver info: driver.version: RemoteWebDriver
        at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(Driv
erCommandExecutor.java:92)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.ja
va:548)
        at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.
java:276)
        at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.ja
va:83)
        at com.ssts.example.IECrashIssue2.main(IECrashIssue2.java:34)
Caused by: java.net.SocketException: Connection reset
        at java.net.SocketInputStream.read(Unknown Source)
        at java.net.SocketInputStream.read(Unknown Source)
        at okio.Okio$2.read(Okio.java:140)
        at okio.AsyncTimeout$2.read(AsyncTimeout.java:237)
        at okio.RealBufferedSource.indexOf(RealBufferedSource.java:355)
        at okio.RealBufferedSource.readUtf8LineStrict(RealBufferedSource.java:22
7)
        at okhttp3.internal.http1.Http1Codec.readHeaderLine(Http1Codec.java:215)

        at okhttp3.internal.http1.Http1Codec.readResponseHeaders(Http1Codec.java
:189)
        at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInter
ceptor.java:88)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:147)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInter
ceptor.java:45)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.ja
va:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.j
ava:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndF
ollowUpInterceptor.java:126)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorCha
in.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient
.java:105)
        at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExe
cutor.java:155)
        at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(Driv
erCommandExecutor.java:83)
        ... 4 more`

**Comment by jimevans** [comment]
Please provide a log from IEDriverServer.exe from the error. I can not reproduce your result with the specified binary using the code you provided.

**Comment by justnatural** [comment]
Hi! 
I don't really think that this error is the one I was encountering, but I was encountering "org.openqa.selenium.SessionNotCreatedException: Unable to create new service: InternetExplorerDriverService" while using Selenium Standalone Server 3.12.0 and Internet Explorer Driver Server 3.12.0. 
Several users were encountering this same issue, but it seemed the moderators did not want to reopen the thread here: https://github.com/SeleniumHQ/selenium/issues/3456.
When I saw that you upgraded to 3.13.0, I downloaded both of them and it worked! 
@jimevans Thanks a lot!
